platforms/hyperledger-indy/charts/indy-node/templates/statefulset.yaml

```diff
@@ -217,7 +219,7 @@ spec:
             - containerPort: {{ $.Values.client.port }}
           env:
             - name: INDY_NODE_NAME
-              value: "{{ $.Values.node.name }}"
+              value: "{{ $.Values.vault.nodeId }}"
```

Comment:
Where is vault.nodeId defined?

Reply:
It is defined in the node.tpl (`platforms/hyperledger-indy/configuration/roles/create/helm_component/node/templates/node.tpl`) as `nodeId: {{ stewardItem.name }}`.